### DISCUSSION TITLE
Add option to display text and value on autocomplete

### DISCUSSION THREAD

**Discussion by dpalyov** [directus/directus#14002] [Feature Requests]
I couldn't find anything on the topic, but wouldn't it be nice if we could somehow concatenate text and value of the selected option
```{{ text }} - {{ value }}``` or vice-versa ? 
To give you a proper example, I am currently facing the situation where my value should be an ```id``` coming from an API. However, once selected input`s value is showing a simple number which is kind of frustrating for the user

**Comment by rijkvanzanten** [comment]
This could work during the initial selection, but on subsequent visits to the page, the app wouldn't have the available data to render the preview (as only the value is stored, which would be the id in this case)

**Comment by dpalyov** [comment]
``<v-list-item v-for="result of results" :key="result.value" @click="() => emitValue(concatTextValue ? `${result.value} - ${result.text}` : result.value)">
      <v-list-item-content>{{ textPath ? result.text : result.value }}</v-list-item-content>
</v-list-item>``

The above is working fine for me on a custom extension. It retains the value as it is.

**Reply by rijkvanzanten** [reply]
Ah, but in this case, you're no longer saving just the ID, you're saving `{id} - {text}` which may or may not be what's required

**Reply by dpalyov** [reply]
yes, thats the intend. That way I assure the user has a human readable value associated withe the id. ``IDs`` only mean nothing when you come back to the page and they are needed for consumption by another client. Probably its too niche as a case, however I think its worth discussing

**Reply by rickysullivan** [reply]
I've run into a similar issue and created a custom interface (forked from the built-in one) for it: https://github.com/directus/directus/discussions/10128#discussioncomment-4649071

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
